crates/sp-domains-fraud-proof/src/tests.rs

```diff
@@ -170,8 +170,7 @@ async fn benchmark_bundle_with_evm_tx(
     const TX_TYPES: u32 = 4;
     let mut thread_rng = rand::thread_rng();
     let between = Uniform::from(0..TX_TYPES);
-    let (slot, bundle) = ferdie.produce_slot_and_wait_for_bundle_submission().await;
-    assert!(bundle.is_some());
```

Comment:
confused with this change. Dont we want to ensure the bundle is submitted anyway?

Reply:
Yeah, but `produce_slot_and_wait_for_bundle_submission` will panic instead if bundle is not submitted after a few tries, so we don't need the `assert` here.